demos/FiniteVolume/CMakeLists.txt

```diff
@@ -25,7 +25,8 @@ burgers_mra.cpp:finite-volume-burgers-mra
 burgers_os.cpp:finite-volume-burgers-os
 )
 
-include(FindPkgConfig)
+include(CMakeFindDependencyMacro)
+find_dependency(PkgConfig)
```

Comment:
Why not use the `SAMURAI_WITH_PETSC` option here and remove 

```
find_dependency(PkgConfig)
pkg_check_modules(PETSC PETSc)
```

that are normally already done.